Does not work with yarn workspaces

```
yarn run v1.15.2
$ react-scripts -r @cypress/instrument-cra start
internal/modules/cjs/loader.js:651
    throw err;
    ^

Error: Cannot find module '/monorepo/packages/my-react-app/node_modules/react-scripts/config/webpack.config.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:649:15)
    at Function.Module._load (internal/modules/cjs/loader.js:575:25)
    at Module.require (internal/modules/cjs/loader.js:705:19)
    at require (internal/modules/cjs/helpers.js:14:16)
    at Object.<anonymous> (monorepo/node_modules/@cypress/instrument-cra/index.js:17:24)
    at Module._compile (internal/modules/cjs/loader.js:799:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:810:10)
    at Module.load (internal/modules/cjs/loader.js:666:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:606:12)
    at Function.Module._load (internal/modules/cjs/loader.js:598:3)
error Command failed with exit code 1.
```

This is looking for `node_modules/react-scripts/config/webpack.config.js`, but this file would actually exist higher up where `yarn workspaces` uses a single `node_modules` for all workspaces that use the same version of `react-scripts` as a dependency.

--- a/index.js
+++ b/index.js
@@ -1,7 +1,9 @@
 const debug = require('debug')('instrument-cra')
 const path = require('path')
+const findYarnWorkspaceRoot = require('find-yarn-workspace-root')
+
 const webpackConfigPath = path.resolve(
-  process.cwd(),
+  findYarnWorkspaceRoot() || process.cwd(),
   'node_modules',
   'react-scripts',
   'config',

--- a/package.json
+++ b/package.json
@@ -31,7 +31,8 @@
   "homepage": "https://github.com/cypress-io/instrument-cra#readme",
   "dependencies": {
     "babel-plugin-istanbul": "5.2.0",
-    "debug": "4.1.1"
+    "debug": "4.1.1",
+    "find-yarn-workspace-root": "^1.2.1"
   },
   "devDependencies": {
     "react": "16.12.0",

